examples/platform/silabs/BaseApplication.cpp

```diff
@@ -24,20 +24,21 @@
 #include "AppConfig.h"
 #include "AppEvent.h"
 #include "AppTask.h"
-
 #include <app/server/Server.h>
 
+#define APP_ACTION_BUTTON 1
+
 #ifdef DISPLAY_ENABLED
 #include "lcd.h"
 #ifdef QR_CODE_ENABLED
 #include "qrcodegen.h"
 #endif // QR_CODE_ENABLED
 #endif // DISPLAY_ENABLED
 
-#include "SilabsDeviceDataProvider.h"
 #if CHIP_CONFIG_ENABLE_ICD_SERVER == 1
 #include <app/icd/server/ICDNotifier.h> // nogncheck
 #endif
+#include <ProvisionManager.h>
```

Comment:
```suggestion
#include <platform/silabs/ProvisionManager.h>
```
Just to be inline wiht the include structure

Reply:
Not sure why, but it doesn't work. With the path the include throws an error. I reverted the change.

Reply:
That is really weird... In theory it should?